src/out

```diff
@@ -91,6 +93,7 @@ body=$(jq -cn \
   --arg       timestamp "${timestamp}000" \
   --arg       severity "$severity" \
   --arg       build_url "$concourse_url/builds/$BUILD_ID" \
+  --arg       build_url_html "<a href=\"$concourse_url/builds/$BUILD_ID\">build_url</a>" \
```

Comment:
just `build_url</a>` here?

Reply:
Maybe `s/build_url/Link to Build`?

Reply:
then, shouldn't `Link to Build` be configurable?

Reply:
Chaned it to Link to  Build, i think its better, and its only for the grafana annotations to be able to be clickable